docker/deployed/staging/api/config.json

```diff
@@ -7,7 +7,7 @@
     "TLSKey": "${VALIDATOR_TLS_KEY}"
   },
   "Gateway": {
-    "ExternalURIPrefix": "https://staging.tableland.network/tables"
```

Comment:
`/tables` is removed because it's part of the [Sprintf](https://github.com/tablelandnetwork/go-tableland/pull/199/files?diff=unified&w=0#diff-495917a0bcbd30f16dabee67340414731bcf65805956d9209a21da47227885e6R66) now.